.github/workflows/nightly.yml

```
@@ -122,31 +75,31 @@ jobs:
   admin:
     uses: ./.github/workflows/admin.yml
     secrets: inherit
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_protected
   integration:
     uses: ./.github/workflows/integration.yml
     secrets: inherit
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_protected
     with:
       nightly: true
   php:
     uses: ./.github/workflows/php.yml
     secrets: inherit
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_protected
   storefront:
     uses: ./.github/workflows/storefront.yml
     secrets: inherit
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_protected
   downstream:
     uses: ./.github/workflows/downstream.yml
     secrets: inherit
-    if: github.repository == 'shopware/shopware'
+    if: github.repository == 'shopware/shopware' && github.ref_protected
```

```suggestion
    if: github.repository == 'shopware/shopware' && github.ref_protected
```

Why do we need this? For testing stuff it's useful to trigger all the workflows with one dispatch

---

TBH, I don't remember why this was needed. I suppose it's a leftover from trying to add the new build process to `shopware/shopware` instead of `shopware/acceptance-test-suite`. I'll remove the conditions.